### DISCUSSION TITLE
Trying to make `curl -F` or form data work

### DISCUSSION THREAD

**Discussion by smokeyhallow** [jnunemaker/httparty#768] [Q&A]
Can anyone please help me implement the following `curl` command in `HTTParty`?

` curl -X POST -F "image=@512x512.png" https://api.example.com/upload`

I tried searching everywhere, but weirdly such an example doesn't even exist in examples list! I wonder how can that be? Is this feature not available in the gem?

This is one of the variations I tried:

`HTTParty.post(URL, headers: headers, form_data: { image: File.read("512x512.png") })`

**Comment by eegnom1807** [comment]
Try with this:
```
def  prueba
        path = "512x512.png"
        url = "https://api.example.com/upload"
        token = "111.qqq.www"
        headers = {
            "Authorization" => "Bearer #{token}"
        }
        
       response = HTTParty.post(
            url, 
            multipart: true, 
            body: { file: File.open(path) }, headers: headers
        )
        puts response
end
```
